<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <comment>This widget is used for creating single dated transfers.
All outstanding single dated transfers are shown in a list</comment>
 <class>GuiDatedTransfersBase</class>
 <widget class="QWidget" name="GuiDatedTransfersBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GuiDatedTransfersBase</string>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="textLabelTitle">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&lt;font size=&quot;+2&quot;&gt;&lt;u&gt;Dated Transfers:&lt;/u&gt;&lt;/font&gt;</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelUpdate">
       <property name="text">
        <string>Last Update of List:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelUpdateDate">
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout">
       <item row="0" column="1">
        <widget class="QLabel" name="textLabelAccounts">
         <property name="text">
          <string>&lt;p align=&quot;right&quot;&gt;Account:&lt;/p&gt;</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="2" colspan="2">
        <widget class="QLabel" name="textLabelAccount">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="buttonNewDatedTransfer">
       <property name="text">
        <string>&amp;New Dated Transfer</string>
       </property>
       <property name="shortcut">
        <string>Alt+N</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonDeleteDatedTransfer">
       <property name="text">
        <string>&amp;Delete Dated Transfer</string>
       </property>
       <property name="shortcut">
        <string>Alt+D</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>61</width>
         <height>21</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonGetList">
       <property name="text">
        <string>&amp;Get List of Dated Transfers Until:</string>
       </property>
       <property name="shortcut">
        <string>Alt+U</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEdit"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTableWidget" name="tableWidget">
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <column>
      <property name="text">
       <string>Status</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Bank Message</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Type</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Execution Date</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Value</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Purpose</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Bank Code</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Account Id</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Last Execution</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Customer Reference</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Id</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections>
  <connection>
   <sender>buttonGetList</sender>
   <signal>clicked()</signal>
   <receiver>GuiDatedTransfersBase</receiver>
   <slot>slotGet()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonDeleteDatedTransfer</sender>
   <signal>clicked()</signal>
   <receiver>GuiDatedTransfersBase</receiver>
   <slot>slotDeleteDatedTransfer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonNewDatedTransfer</sender>
   <signal>clicked()</signal>
   <receiver>GuiDatedTransfersBase</receiver>
   <slot>slotNewDatedTransfer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
